src/components/Home/BannerStandard.tsx

```diff
@@ -0,0 +1,41 @@
+interface BannerInfo {
+  img: string;
+  title: string;
+  desc: string;
+  button: string;
+}
+
+function BannerStandard({ img, title, desc, button }: BannerInfo) {
+  return (
+    <div
+      className="flex w-3/4 min-w-[200px] max-w-[440px] md:w-4/5 md:min-w-[825px] md:max-w-[1100px] md:h-[505px]
+        flex-col md:flex-row-reverse md:justify-between items-center gap-2.5 md:gap-0">
+      <div className="md:w-[440px] md:h-[505px] shrink-0 self-stretch">
+        <img src={img} />
+      </div>
+
+      <div className="flex flex-col items-center md:items-start gap-5 self-stretch md:self-auto">
+        <div>
+          <p
+            id="hoodie-title"
+            className="self-stretch md:self-auto text-center md:text-left text-4xl md:text-5xl font-bold">
+            {title}
+          </p>
+        </div>
+
+        <div id="hoodie-desc" className="md:w-3/4">
+          <p className="self-stretch md:self-auto text-center md:text-left text-base font-medium">{desc}</p>
+        </div>
+
+        <div
```

Comment:
Wrap with a <Link> from [react router](https://reactrouter.com/en/main/components/link) or replace div with link tag. Also add the link url to props

Reply:
Did this but now the button's text has default colours :(